service/src/main/java/bio/terra/pipelines/service/PipelineRunsService.java

```diff
@@ -93,21 +96,29 @@ public Map<String, Map<String, String>> preparePipelineRun(
       throw new InternalServerErrorException("%s workspaceId not defined".formatted(pipelineName));
```

Comment:
i was looking at this and we should probably change the `name` column of Pipelines to be auto converted to a PipelinesEnum similar to how we treat the `type` column in the PipelineInputDefinitions table.  We do a lot of conversions back and forth throughout this file and it would just be clearner.  Maybe worth make a ticket for?

Reply:
great idea. https://broadworkbench.atlassian.net/browse/TSPS-263